### ISSUE TITLE
Month dropdown is disabled once minYear is added

### ISSUE THREAD

**Issue by parminderjit-s** [dangrossman/daterangepicker#2111]
`   
$('.daterange-single-startYear').daterangepicker({ 	   
	        singleDatePicker: true,
	        showDropdowns: true,
	        minYear: 1980,       
	    });
`

![date](https://user-images.githubusercontent.com/58060832/84640305-13925380-af2c-11ea-81a6-da62e995dd13.jpg)

**Comment by parminderjit-s** [comment]
I fixed it by placing a  minDate: '01/01/1980',

Is this intended this way or is this a bug?

**Comment by AcidRaZor** [comment]
No, I think this is a bug, see https://github.com/dangrossman/daterangepicker/issues/1974 reported over a year ago.

Note that placing minDate (or maxDate) needs to match the locale format (if you're using moment to generate the dates), otherwise it doesn't work either.

**Comment by jonathanloos** [comment]
+1 on this!

**Comment by ethansnow2012** [comment]
I was having the same problem. I fixed it by setting minYear and maxYear by string instead of integer.

Hope this helps.
